loginas/settings.py

```diff
@@ -17,6 +17,8 @@
 
 MESSAGE_LOGIN_REVERT = getattr(
     settings,
-    "MESSAGE_LOGIN_REVERT",
+    "LOGINAS_MESSAGE_LOGIN_REVERT",
     _("You are now logged back in as {username}")
 )
+
+MESSAGE_EXTRA_TAGS = getattr(settings, 'LOGINAS_MESSAGE_EXTRA_TAGS', '')
```

Comment:
Why not call this `LOGINAS_MESSAGE_EXTRA_TAGS` when you just renamed the above to be consistent?